oceannavigator/frontend/src/components/MapInputs.jsx

```diff
@@ -283,24 +222,15 @@ MapInputs.propTypes = {
   state: PropTypes.object,
   sidebarOpen: PropTypes.bool,
   basemap: PropTypes.string,
-  scale: PropTypes.string,
-  scale_1: PropTypes.string,
   bathymetry: PropTypes.bool,
   dataset_compare: PropTypes.bool,
-  dataset_1: PropTypes.object,
   projection: PropTypes.string,
-  depth: PropTypes.number,
-  time: PropTypes.number,
-  variable_scale: PropTypes.array,
   extent: PropTypes.array,
   changeHandler: PropTypes.func,
   swapViews: PropTypes.func,
   showHelp: PropTypes.func,
   options: PropTypes.object,
   updateOptions: PropTypes.func,
-  availableDatasets: PropTypes.arrayOf(PropTypes.object),
-  datasetVariables: PropTypes.arrayOf(PropTypes.object),
-  datasetDepths: PropTypes.arrayOf(PropTypes.object)
```

Comment:
Remove a bunch of unused crap from the props and proptypes across several files.